app/models/difficulty_level.rb

```diff
@@ -3,4 +3,5 @@ class DifficultyLevel < ActiveRecord::Base
   has_many :events, dependent: :nullify
 
   validates :title, presence: true
+  validates :color, format: /\A#[0-9a-fA-F]{6}\z/
```

Comment:
The reason why  I put that validation here is that if there is not color or it doesn't have the right format what I have done do not always work properly, and the tests fail.

So, I am not sure if I have to leave it here, move to the next commit or move it to a new commit.